### ISSUE TITLE
Separate sorting settings for every playlist

### ISSUE THREAD

**Issue by Golem642** [Malopieds/InnerTune#217]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

I would like to be able to sort some playlist by date and some other by personal preference without have to change the sorting method everytime i enter one.
So if i change the sorting setting inside a playlist, it won't get modified for every playlist but just for this one. Also have a default sorting mode in settings for people who needs it

### Why do you want this feature?

I like to sort my playlist differently, for example I keep the playlist I sync with YouTube sorted by newest entry. But, i sort some other playlists i made in the app by personal preference. It's annoying for me when i have to change the sorting mode every time i enter a playlist and i want to find a song

**Comment by Golem642** [comment]
Also side thing I noticed, the "by date" sort doesn't work greatly in the YT music playlist for some reason ?
It worked until some point but my now my most 3 recently added songs are correctly at the top and the rest is sorted by oldest first, so the most recent songs after those 3 are at the very bottom of the list.
I don't know why it's doing this

**Comment by Malopieds** [comment]
I don't think I'll do this one. The reason is it would mean modify the whole database just to do something that take 3 clicks.

> doesn't work greatly in the YT music playlist for some reason ?

Did you synced your playlist?

**Comment by Golem642** [comment]
> Did you synced your playlist?

Yes and it doesn't change anything. New songs are added at the top but old ones are sorted the wrong way around
